server/src/internalClusterTest/java/org/opensearch/remotestore/RemoteStoreRestoreIT.java

```diff
@@ -453,4 +468,47 @@ public void testRTSRestoreNoData() throws IOException {
     }
 
     // TODO: Restore flow - index aliases
```

Comment:
Currently we can restore aliases as well right as that information is present in the IndexMetadata itself.

Reply:
Created and issue for this - https://github.com/opensearch-project/OpenSearch/issues/9814.